public/scripts/dom-utils.js

```diff
@@ -19,6 +19,16 @@ export function toggleVisibleElement($el) {
   }
 }
 
+export function sortChildren($el, attribute) {
+  [...$el.children]
+    .sort(
+      (a, b) =>
+        parseInt(a.getAttribute(attribute)) -
+        parseInt(b.getAttribute(attribute)),
+    )
+    .forEach((node) => $el.appendChild(node));
+}
```

Comment:
Here is a improved async version of this function to do the ordering instantly:

```suggestion
  const sorted = [...$el.children]
    .sort(
      (a, b) =>
      parseInt(a.getAttribute(attribute)) -
      parseInt(b.getAttribute(attribute)),
    )
  $el.innerHTML = ""
  sorted.slice(0, 15).forEach((node) => $el.appendChild(node))
  setTimeout(async () => {
    sorted.slice(15).forEach((node) => $el.appendChild(node));
  }, 5);
```

Resetting the HTML content and appending synchronously only the first 15 icons do the trick. I suppose that this number can be increased without decreasing performance.